runtime/metrics.rs

```diff
@@ -108,7 +146,8 @@ pub fn metrics_op(op_fn: Box<OpFn>) -> Box<OpFn> {
         let fut = fut
           .inspect(move |buf| {
             let mut s = op_state_.borrow_mut();
-            let metrics = s.borrow_mut::<Metrics>();
+            let runtime_metrics = s.borrow_mut::<RuntimeMetrics>();
+            let metrics = runtime_metrics.ops.get_mut(name).unwrap();
```

Comment:
Is it possible to key ops by OpId rather than by name?
Seems more efficient to do it that way.

Reply:
See my comment in the PR description:

> This could be improved by keying the metrics hashmap using the u64 op ID instead of a string like we do now. This would require that the calling op id is somehow exposed to the metrics exporter though, which is currently not the case.